Policies/FetchMetadataDefaultPolicy.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Ise\WebSecurityBundle\Policies;
+use Ise\WebSecurityBundle\Policies\FetchMetadataPolicyInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class FetchMetadataDefaultPolicy implements FetchMetadataPolicyInterface
+{
+    /**
+     * CorsEndpoints denotes the allowed cross origin endpoints as part of the Default Fetch metadata policy
+     *
+     * @var array
+     */
+    private $corsEndpoints;
```

Comment:
What's the format for this? Array of URL string?

Reply:
Yes, I tried to set it to a more...reasonable identifier like a string array. But I ended up getting a number of angry responses from Symfony's auto-wire service. It's something I'll continue to iterate with. Until then though I'll update that with an example for the input, and then merge this into main.